msprime/ancestry.py

```diff
@@ -1078,6 +1079,26 @@ def _parse_sim_ancestry(
     random_seed = _parse_random_seed(random_seed)
     random_generator = _msprime.RandomGenerator(random_seed)
 
+    if stop_at_local_mrca is None:
+        stop_at_local_mrca = True
+    elif not isinstance(stop_at_local_mrca, bool):
```

Comment:
We probably don't need to force it to be a bool (unless that's what we do with other params). Something like
```
if stop_at_local_mrca is None:
    stop_at_local_mrca = True
if not stop_at_local_mrca:
    if end_time == math.inf or end_time is None:
           ValueError()
```